src/commands/appSettings/connectionSettings/getLocalConnectionSetting.ts

```diff
@@ -21,11 +21,17 @@ export interface IResourceResult {
     connectionString: string;
 }
 
-export async function getStorageConnectionString(context: IStorageAccountWizardContext): Promise<IResourceResult> {
+export async function getStorageConnectionString(context: IStorageAccountWizardContext & { useStorageEmulator?: boolean }): Promise<IResourceResult> {
+    if (context.useStorageEmulator) {
+        return {
+            name: randomUtils.getRandomHexString(6),
```

Comment:
I think I prefer Eric's suggestion where we do random hex string + `_STORAGE`. I think it's worth adding the label on there so it's more clear what this is for.

Reply:
It does do that still. It's confusing, but it happens here: https://github.com/microsoft/vscode-azurefunctions/pull/3924/files/ab23bf1afd5053a427989abb73574fbc13ad9094#diff-50e86af002b096abb16ed6118f7678e5f2a914a1985c9c4706b8674a77047c87R39